src/main/java/uk/gov/justice/digital/delius/jpa/standard/repository/StaffRepository.java

```diff
@@ -30,4 +30,11 @@
 
     @Query("select s.staffId from Staff s where s.officerCode = :officerCode")
     Optional<Long> findStaffIdByOfficerCode(String officerCode);
+
+    @Query("select staff " +
+        "from Staff staff, StaffTeam staffTeam " +
+        "where staffTeam.staffId = staff.staffId " +
+        "and staffTeam.teamId = :teamId " +
+        "and (staff.endDate is null or staff.endDate > CURRENT_DATE) ")
```

Comment:
Java now has text blocks (i.e. multi-line strings) 😄

Might be cleaner to do this instead of using concatenation?
```suggestion
    @Query("""
        select staff
         from Staff staff, StaffTeam staffTeam
         where staffTeam.staffId = staff.staffId
         and staffTeam.teamId = :teamId
         and (staff.endDate is null or staff.endDate > CURRENT_DATE)
        """)
```